core-java-modules/core-java-uuid/src/test/resources/logback.xml

```diff
@@ -12,6 +12,7 @@
 
     <!-- in order to debug some marshalling issues, this needs to be TRACE -->
     <logger name="org.springframework.web.servlet.mvc" level="WARN" />
+    <logger name="c.b.u.UUIDPositiveLongGeneratorUnitTest" level="INFO"/>
```

Comment:
This is redundant because the test only uses INFO logging, and INFO is the default in this file too.

When we set INFO level here, it means "show logs from this class for ERROR, WARN, INFO but not for DEBUG and TRACE".
```suggestion
```